_data/data.yml

```diff
@@ -462,13 +469,67 @@ people:
           photo: "img/people/kristel_fung.jpg"
           interests: "Cybersecurity, cryptography, software development"
 
+      - &yuanrui_chen
+          name: "Yuanrui Chen"
+          anchor: yuanrui_chen
+          internal: true
+          role: "Undergraduate student"
+          since: 2020
+          photo: "img/people/yuanrui_chen"
+          interests: "Network security, software security, and software development"
+
+      - &isha_dave
+          name: "Isha Vipul Dave"
+          anchor: isha_dave
+          internal: true
+          role: "Undergraduate student"
+          since: 2020
+          photo: "img/people/isha_dave"
+          interests: "Cybersecurity and game design and development"
+
+      - &kaitlyn_liu
+          name: "Kaitlyn Liu"
+          anchor: kaitlyn_liu
+          internal: true
+          role: "Undergraduate student"
+          since: 2020
+          photo: "img/people/kaitlyn_liu"
+          interests: "Systems security and computer vision"
+
+      - &benjamin_wu
+            name: "Benjamin Wu"
+            anchor: benjamin_wu
+            internal: true
+            role: "Undergraduate student"
+            since: 2020
+            photo: "img/people/benjamin_wu"
+            interests: "Financial technology, cybersecurity, cryptography"
+
+      - &ge_yang
+            name: "Ge Yang"
+            anchor: ge_yang
+            internal: true
+            role: "Undergraduate student"
+            since: 2020
+            photo: "img/people/ge_yang"
+            interests: "Security and privacy"
+
+    - &henry_zhou
```

Comment:
It says there is an error on this line https://travis-ci.org/github/secure-systems-lab/ssl-site/builds/673149908

I think your indentation is off which is causing the issue.

Reply:
I may have diagnosed the problem. II thought I had accidentally removed Henry, so I re-added him. Lets see if this works.